subprojects/core-api/src/main/java/org/gradle/api/initialization/dsl/DependenciesModelBuilder.java

```diff
@@ -129,5 +129,12 @@
          * @param versionRef the version reference
          */
         void versionRef(String versionRef);
+
+        /**
+         * Do not associate this alias to a particular version, in which
+         * case the dependency notation will just have group and artifact.
+         *
+         */
+        void withoutVersion();
```

Comment:
I wish that was not needed and you could stop the builder, omitting that call.

Reply:
We can probably get rid of this by keeping track of dangling builders. However I'm not sure _when_ to collect the dangling builders...